DocumentsFromSnapshotMigration/src/test/java/org/opensearch/migrations/bulkload/EndToEndTest.java

```diff
@@ -179,4 +180,29 @@ private void checkDocsWithRouting(
         }
     }
 
+    // private static Stream<Arguments> scenarios_experimental() {
+    //     var scenarios = Stream.<Arguments>builder();
+
+    //     for (var sourceCluster : SupportedClusters.sources()) {
+    //         for (var targetCluster : SupportedClusters.targets_experimental()) {
+    //             scenarios.add(Arguments.of(sourceCluster, targetCluster));
+    //         }
+    //     }
+
+    //     return scenarios.build();
+    // }
+
+    // @ParameterizedTest(name = "Source {0} to Target {1}")
+    // @MethodSource(value = "scenarios_experimental")
+    // public void migrationDocumentsExperimental(
+    //     final SearchClusterContainer.ContainerVersion sourceVersion,
+    //     final SearchClusterContainer.ContainerVersion targetVersion) throws Exception {
+    //     try (
+    //         final var sourceCluster = new SearchClusterContainer(sourceVersion);
+    //         final var targetCluster = new SearchClusterContainer(targetVersion)
+    //     ) {
+    //         migrationDocumentsWithClusters(sourceCluster, targetCluster);
+    //     }
+    // }
+
```

Comment:
Is all of this intended to be in this PR?

Reply:
Removed